package-lock.json

```diff
@@ -4,6 +4,19 @@
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
```

Comment:
Do we need to update the package-lock.json? If you say we do then I'm up for it. I just want to make sure this is 
a conscious decision because if I remember correctly we had some problems in the past with multiple OS compatibility due to the package-lock.

Reply:
Yeah, these changes are needed since some dependencies were added/updated compared to old lock file.